woltka/coverage.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020--, Qiyun Zhu.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+"""Functions for calculating coverage of subject sequences.
+
+The "coverage" refers to the ranges of a subject sequence (e.g., genome) that
+are covered by at least one query sequence (e.g., short sequencing read).
+"""
+
+from os import makedirs
+from os.path import join
+from collections import deque
+
+
+def merge_ranges(ranges):
+    """Merge short, fragmental ranges into long, continuous ones.
```

Comment:
Can my name be credited somewhere as the source of the algorithm used in this function?

Reply:
Sure! I appreciate your contribution! I have added the credit to the docstring of `coverage.py`. How does it read?